config/clusters/templates/gcp/cluster.yaml

```diff
@@ -0,0 +1,29 @@
+name: {{ cluster_name }}
+provider: gcp
+gcp:
+  key: enc-deployer-credentials.secret.json
+  project: {{ project_id }}
+  cluster: {{ cluster_name }}
+  # We default to a regional cluster
+  zone: {{ cluster_region }}
+support:
+  helm_chart_values_files:
+    - support.values.yaml
+    - enc-support.secret.values.yaml
+hubs:
+  - name: {{ hub_name }}
+    # Tip: consider changing this to something more human friendly
+    display_name: "{{ cluster_name }} - {{ hub_name }}"
+    domain: {{ hub_name }}.{{ cluster_name }}.2i2c.cloud
+    helm_chart: {{ hub_type }}
+    auth0:
+      # Tip: Uncomment the line below and enable auth0
```

Comment:
Let's remove this by comment by default, as we don't really wanna use auth0 for anything new.